designs/2022-community-eslint-org/README.md

```diff
@@ -0,0 +1,263 @@
+- Repo: No ESLint repos, only the ones mentioned below (owned by private
+  persons)
+- Start Date: 2022-05-31
+- RFC PR:
+- Authors: [@aladdin-add](https://github.com/aladdin-add),
+  [@MichaelDeBoey](https://github.com/MichaelDeBoey),
+  [@ota-meshi](https://github.com/ota-meshi) &
+  [@voxpelli](https://github.com/voxpelli)
+
+# `@eslint-community` GitHub organisation
+
+## Summary
+
+<!-- One-paragraph explanation of the feature. -->
+
+Facilitate a GitHub organisation
+([@eslint-community](https://github.com/eslint-community)) where community
+members could maintain widely used ESLint related packages that would otherwise
+get abandoned.
+
+## Motivation
+
+<!-- Why are we doing this? What use cases does it support? What is the expected
+outcome? -->
+
+There a some widely used ESLint related packages that aren't actively maintained
+anymore and we'd like to transfer them to a community owned GitHub organization
+(`@eslint-community`), so we can give them back to the community.
+
+## Detailed Design
+
+<!--
+   This is the bulk of the RFC.
+
+   Explain the design with enough detail that someone familiar with ESLint
+   can implement it by reading this document. Please get into specifics
+   of your approach, corner cases, and examples of how the change will be
+   used. Be sure to define any new terms in this section.
+-->
+
+### How do you envision `@eslint-community`?
+
+Just like the [@jest-community](https://github.com/jest-community) org, the
+`@eslint-community` org would be a collection of important/widely used ESLint
```

Comment:
I would prefer `widely depended` upon rather than `important/widely`.

Some modules can be widely used without actually holding the wider community back. The trickier part are plugins like `eslint-plugin-node` and similar, that used to be core ESLint functionality that many people used but now lives on in abandoned modules and eg. held users of them back from upgrading to ESLint 8. Such modules are hard to replace.